app/views.py

```diff
@@ -16,51 +13,44 @@
 def home():
     if "user" in session:
         user = session["user"]
-        return render_template("index.html", user = user)
+        return render_template("index.html", user=user)
```

Comment:
lack of spaces is prefered python style in this context?

Reply:
yes